client/debian_make_all.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+make clean
+sudo apt-get install -y graphicsmagick-libmagick-dev-compat libavcodec-dev libavformat-dev libswscale-dev
```

Comment:
why you use the  `graphicsmagick-libmagick-dev-compat` instead of the `libgraphicsmagick++-dev` from the [README](https://github.com/hzeller/flaschen-taschen/blob/master/client/README.md) ?

Reply:
Done!